browser/main/SideNav/SideNav.styl

```diff
@@ -53,11 +30,33 @@
   display flex
   flex-direction column
 
-.tag-title
-  padding-left 15px
-  padding-bottom 13px
-  p
-    color $ui-button-default-color
+.tag-control
+  display flex
+  height 30px
+  line-height 25px
+  overflow hidden
```

Comment:
Is this the correct way to hide the selector when the sidebar is too narrow?

Reply:
I think so. :)